api/lib/spree/api/responders/jbuilder_template.rb

```diff
@@ -3,7 +3,7 @@
 module Spree
   module Api
     module Responders
-      module RablTemplate
+      module JbuilderTemplate
```

Comment:
I think we should keep the `RablTemplate` class here and deprecate it since it could be used in host applications and it's just matter of adding a `DeprecatedConstantProxy`:

```ruby
RablTemplate = ActiveSupport::Deprecation::DeprecatedConstantProxy.new('RablTemplate', 'JbuilderTemplate')
```

 and a spec that ensures it's working as expected.

Reply:
Updated 😄 what do you think?